piper/ratbagd.py

```diff
@@ -420,6 +420,9 @@ def set_default(self):
 class RatbagdButton(_RatbagdDBus):
     """Represents a ratbagd button."""
 
+    KEY_PRESS = 0
+    KEY_RELEASE = 1
```

Comment:
actually, no. libratbag doesn't use that, if you press+release, that is two events in the macro. this would have to be abstracted in the next layer, imo.